Sources/DeepLearning/Operators.swift

```diff
@@ -16,12 +16,12 @@
 import TensorFlow
 #endif
 
-// Rounds the values of a tensor to the nearest integer, element-wise.
+/// Returns the values of the specified tensor to the nearest integer, element-wise.
```

Comment:
```suggestion
/// Returns the values of the specified tensor rounded to the nearest integer, element-wise.
```

Reply:
Oops, thanks! Fixed in https://github.com/tensorflow/swift-apis/pull/46/commits/e4985e91fbade933588d1f685e4fa4321f3e303b.